test/parallel/xyk-pallet.API.mintLiquidity.error.test.ts

```diff
@@ -57,6 +63,8 @@ describe("xyk-pallet - Mint liquidity tests: MintLiquidity Errors:", () => {
     keyring.addPair(testUser1.keyRingPair);
     keyring.addPair(sudo.keyRingPair);
     await testUser1.addGASPTokens(sudo);
+    const currency = await SudoDB.getInstance().getTokenId();
+    expect(currency).not.toEqual(undefined);
```

Comment:
do not fully understand the reason for this change,

Reply:
It's my mistake; this is useful here. Fixed